contracts/staking/TokenStaking.sol

```diff
@@ -734,8 +736,10 @@ contract TokenStaking is Ownable, IStaking, Checkpoints {
         );
         operatorStruct.tStake -= amount;
         require(
-            operatorStruct.tStake >= minTStakeAmount ||
-                operatorStruct.startTStakingTimestamp + MIN_STAKE_TIME <=
+            operatorStruct.nuInTStake != 0 ||
+                operatorStruct.keepInTStake != 0 ||
```

Comment:
Does this mean that it's possible to satisfy the entire require by leaving a small amount of NU/KEEP?

Reply:
yep, because staker can change operator only if there is no more staked nu, so 10^-18 Nu will prevent from using other operator